fotoapparat/src/main/java/io/fotoapparat/Fotoapparat.java

```diff
@@ -250,6 +269,11 @@ public Fotoapparat autoFocus() {
         return autoFocusRoutine.autoFocus();
     }
 
+    public void setTapToFocusEnabled(boolean isEnabled) {
```

Comment:
I believe it is not a responsibility of Fotoapparat to enabled or disable tap-to-focus feature. Instead, I would imagine that the app which uses the library would just not trigger manual focus if it does not want to.

Reply:
But what if we would like to enable/disable tap-to-focus during our interaction with CameraView? Switching Fotoapparat instances seems to be overhead for this.

Reply:
Maybe I did not yet understand the logic fully. I will finish review and then see if this comment still applies.